index.html

```diff
@@ -51,7 +51,117 @@ <h3>I'm a Software Developer</h3>
             <li><a href="https://www.linkedin.com/in/philip-modinos-14195021"><i class="fa fa-linkedin-square fa-3x" aria-hidden="true"></i></a></li>
         </ul>
     </aside>   
+    
+</div> <!--page 1 end -->
 
+<div class="page2">
+    <div class="container">
+        <div class="items title1">
+            My Recent Works        
+        <br> <br>
+        <hr>
+        </div>
+        <div class="items img1">
+            <img src="images/testimg.png" alt="">
+        </div>
+        <div class="items description1">
+            <div class="title2">Multi-Post Stories</div>
+            <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+        </div>
+        <ul class="items lang">
+            <li>html</li>
+            <li>css</li>
+            <li>php</li>
+            <li>mysql</li>
+            <li>javascript</li>
+            
+        </ul>
+        <div class="items buttontypeA">
+        <a href="#">See Project</a>
+        </div>
+        <div class="items site1">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site1 end-->
+        <br> <br> <br>
+        <div class="items site1 site2">
+            <div class="description2">
+                <div class="title3">Multi-Post Stories</div>
+                <div class="text1">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</div>
+            </div>
+            <ul class="items lang">
+                <li>html</li>
+                <li>css</li>
+                <li>php</li>
+                <li>mysql</li>
+                <li>javascript</li>
+                
+            </ul>
+            <div class="items buttontypeB">
+                <a href="#">See Project</a>
+                </div>
+        </div><!--site2 end-->
+        
+    </div><!--container end-->
+</div><!--page2 end-->
+<br> <br> <br>
+<div class="page3">
+    <div class="container">
+        <br> <br> 
+        <div class="items title1">About me</div>
```

Comment:
- [ ] As I mentioned earlier, please make sure to apply an `<hx>` tag where a title is required. This is to improve your code to be more semantic.